pos_analytic_by_config/__manifest__.py

```diff
@@ -1,14 +1,16 @@
 # Copyright 2015 ACSONE SA/NV
+# Copyright 2020 Tecnativa - David Vidal
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     'name': "POS Analytic Config",
     'summary': """Use analytic account defined on
                   POS configuration for POS orders""",
     'author': 'ACSONE SA/NV,'
+              'Tecnativa'
```

Comment:
I suppose this is because the migration on v11 was hard and you didn't remember to add it in that moment, isn't it?

Reply:
Yes, that's it :)